thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/dashboard/views/tabs/analysis.ftl

```diff
@@ -1,74 +1,27 @@
 <div class="container-fluid ">
 	<div class="row bg-white row-bordered">
-		<div class="container top-buffer bottom-buffer ">
-			<div class=row>
-				<div class="col-md-8">
-					<div style="float: left;">
-						<label for="metric-input" class="label-large-light">Metric Name: </label>
-					</div>
-					<div style="width: 370px; float: left">
-						<select style="width: 100%" id="analysis-metric-input" class="label-large-light underlined"></select>
-					</div>
-				</div>
-        <div class="col-md-2">
-          <input id="analysis-apply-button" type="button" class="btn btn-info" value="Apply">
-        </div>
-			</div>
-		</div>
-	</div>
-</div>
-<div class="container-fluid">
-	<div class="row bg-white row-bordered">
-		<div class="container top-buffer bottom-buffer">
-			<div class=row>
-
-				<div class="col-md-5">
-					<label>Select time ranges to compare:</label>
-
-					<div class="datepicker-field">
-						<label class="label-description">DATE RANGE(CURRENT) </label>
-						<div id="current-range" class="datepicker-range">
-							<span></span><b class="caret"></b>
-						</div>
-					</div>
+		<div class="container">
+			<div class="search-bar">
 
-					<div class="datepicker-field">
-						<label class="label-description">COMPARE TO(BASELINE)</label>
-						<div id="baseline-range" class="datepicker-range">
-							<span></span> <b class="caret"></b>
-						</div>
-					</div>
-
-				</div>
-				<div class="col-md-2">
-					<label for="granularity">Granularity </label>
-          <select id="analysis-granularity-input" style="width: 100%" class="label-large-light underlined"></select>
-        </div>
-				<div class="col-md-2">
-					<label for="add-dimension-button">Dimensions</label>
-          <select id="analysis-metric-dimension-input" style="width: 100%;" class="label-large-light underlined filter-select-field"></select>
+				<div class="search-title">
+					<label for="metric-input" class="label-large-light">Metric Name: </label>
 				</div>
-				<div class="col-md-3">
-					<label for="add-filter-button">Filters </label>
-          <select id="analysis-metric-filter-input" style="width: 100%;" class="label-large-light underlined"></select>
+
+				<div class="search-input search-field">
+					<select style="width:100%" id="analysis-metric-input" class="label-large-light underlined"></select>
```

Comment:
Unfortunately this is needed because of `select2` component not reading consistently reading the width from a class selector.

`Select2 will do its best to resolve the percent width specified via a css class, but it is not always possible. The best way to ensure that Select2 is using a percent based width is to inline the style declaration into the tag.`